cluster/addons/calico-policy-controller/calico-node-daemonset.yaml

```diff
@@ -103,7 +115,7 @@ spec:
                     {
                       "type": "portmap",
                       "capabilities": {"portMappings": true},
-                      "snat": false
+                      "snat": true
```

Comment:
false right - ip-masq-agent will be used?

Reply:
that's because `"snat": true` only applies to local host traffic. See https://github.com/containernetworking/plugins/tree/master/plugins/meta/portmap#snat

@caseydavenport spotted that if we try to reach hostport from the same node as the hostport pod with `localhost:<hostport>` it won't work without snat: true

This shouldn't affect ip-masq-agent since this is only for the localhost traffic

Reply:
Yep, this is just for the hostport plugin, not general masquerade (which will still be done by the ip-masq-agent).

The hostport plugin needs this setting in order to allow users to do `curl localhost:hostport` successfully.